cmd/sti/main.go

```diff
@@ -10,11 +10,13 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
-	"github.com/openshift/source-to-image/pkg/sti"
-	"github.com/openshift/source-to-image/pkg/sti/api"
-	"github.com/openshift/source-to-image/pkg/sti/config"
-	"github.com/openshift/source-to-image/pkg/sti/errors"
-	"github.com/openshift/source-to-image/pkg/sti/version"
+	"github.com/openshift/source-to-image/pkg/api"
+	"github.com/openshift/source-to-image/pkg/build/strategies"
+	"github.com/openshift/source-to-image/pkg/build/strategies/sti"
```

Comment:
Does it make sense to have `build/strategies`? I'd propose having just `build` or `strategies` package, where the first makes more sense IMHO than the latter.